bocoup/aria-at-report
# Send all testers tests results to the front end

Remaining work:
- [x] make sure you can only see tests in the test queue FOR YOU (if you are a tester....)
- [x] You need to see your own test results as read only in the test run (right now this is broken because I changed data model)
- [x] Count other user's test's progress

This also changes the user queue to show the username instead of the fullname, since that is not always pull from GitHub.

To test!
- Start a test cycle
- Save some results in a test (by filling in data and clicking "next")
- Make sure the test queue accurately shows how many tests you have completed, and test you have complete show up as read only
